gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -295,13 +309,19 @@ The remaining procedures walk you through steps for creating and testing a new H
 +
 image::gateway-quickstart-k8-api-config-endpoint.png["Screenshot of 'Endpoint' page",width=90%]
 +
+.. For the *Scheme*, use the default, *HTTP* setting.
++
+This guide focuses on a simple HTTP API. For HTTPS configuration details, see xref:flex-conn-tls-config.adoc[].
 .. Copy the following URI into the *Implementation URI* field:
 +
 [source]
 ----
 https://jsonplaceholder.typicode.com/
 ----
 .. Type `80` in the *Port* field.
++
```

Comment:
here's the suggested update, below, @seblaz 

please check it out when you have some time. Thx!